netty/src/main/java/io/grpc/netty/HandlerSettings.java

```diff
@@ -23,9 +23,12 @@
 /**
  * Allows autoFlowControl to be turned on and off from interop testing and flow control windows to
  * be accessed. For internal use only.
+ *
+ * @deprecated renamed to {@link InternalHandlerSettings} and should not be used externally
  */
 @VisibleForTesting // Visible for tests in other packages.
 @Internal
+@Deprecated
 public final class HandlerSettings {
```

Comment:
as discussed offline, all public methods in `HandlerSettings` just call `InternalHandlerSettings`'s methods.